freeipa/src/main/java/com/sequenceiq/freeipa/service/operation/OperationService.java

```diff
@@ -99,6 +102,14 @@ public Operation getOperationForAccountIdAndOperationId(String accountId, String
         return operationOptional.get();
     }
 
+    public static void addMdcOperationId(String operationId) {
+        MDCBuilder.addMdcField(OPERATION_ID_MDC_NAME, operationId);
+    }
+
+    public static void removeMdcOperationId() {
+        MDCBuilder.removeMdcField(OPERATION_ID_MDC_NAME);
+    }
```

Comment:
I think you should add these methods to `MDCBuilder` as we have it like `addEnvCrn`